client/src/components/AuthForm.js

```diff
@@ -0,0 +1,171 @@
+                            id="name"
+                            label="Name"
+                            className={classes.textField}
+                            margin="normal"
+                            value={authForm.name}
+                            onChange={handleChange("name")}
+                            variant="outlined"
+                            required
+                            fullWidth
+                        />
+                    ) : (
+                        ""
+                    )}
+                    <TextField
+                        id="email"
+                        label="Email"
+                        type="email"
+                        className={classes.textField}
+                        margin="normal"
+                        value={authForm.email}
+                        onChange={handleChange("email")}
+                        required
+                        variant="outlined"
+                        fullWidth
+                    />
+                </div>
+                <div>
+                    <TextField
+                        id="password"
+                        label="Password"
+                        className={classes.textField}
+                        margin="normal"
+                        type="password"
+                        value={authForm.password}
+                        onChange={handleChange("password")}
+                        required
+                        variant="outlined"
+                        fullWidth
+                        inputRef={pwdRef}
+                    />
+                    {name === "signup" ? (
+                        <TextField
+                            id="confirmPassword"
+                            label="Repeat Password"
+                            className={classes.textField}
+                            margin="normal"
+                            type="password"
+                            value={authForm.confirmPassword}
+                            onChange={handleChange("confirmPassword")}
+                            required
+                            variant="outlined"
+                            fullWidth
+                            inputRef={pwdRepeatRef}
+                        />
+                    ) : (
+                        ""
+                    )}
+                </div>
+                <div>
+                    <Button
+                        variant="contained"
+                        color="primary"
+                        size="small"
+                        type="submit"
+                    >
+                        {name === "signup" ? "Create" : displayName}
+                    </Button>
+                </div>
+            </form>
+        </div>
+    );
+};
+
+export default AuthForm;
+
+/**
+ * PROP TYPES
+ */
+
+// AuthForm.propTypes = {
+//     name: PropTypes.string.isRequired,
+//     displayName: PropTypes.string.isRequired
+//
+// }
```

Comment:
Can probably add these now and not be comments right?

Reply:
yep, added now